### PR TITLE
Update engine and deployment

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,16 +5,17 @@
...
-    "test:schema": "npm run test -- --schema-only",
-    "test:modified": "npm run test -- --modified",
+    "test:schema": "ota validate --schema-only",
+    "test:modified": "ota validate --modified",
     "start": "ota track",
-    "start:api": "ota serve",
+    "start:collection-api": "ota serve",
     "start:schedule": "npm run start -- --schedule",
-    "start:modified": "git diff --name-only HEAD $(git merge-base HEAD origin/main) -- ./declarations/ | grep -v .history | grep -v .filters | sed 's/declarations\\///g' | sed 's/.json//g' | sed 's/.*/\"&\"/' | xargs -r npm run start -- --services",
+    "start:federation-api": "ota-federation-api serve",
```
Comment (clementbiron) [inline package.json:13]:
```suggestion
```

**Review by clementbiron — CHANGES_REQUESTED** [review]
You also need to update the [configuration file](https://github.com/OpenTermsArchive/dating-declarations/blob/main/config/production.json) ;)

**Comment by Ndpnt** [comment]
> You also need to update the [configuration file](https://github.com/OpenTermsArchive/dating-declarations/blob/main/config/production.json) ;)

🙏

**Review by clementbiron — APPROVED** [review]
🚀

**Pull request merged**